recipe/meta.yaml

```diff
@@ -9,30 +8,30 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 0c3c816a028d47f659d6ff5c745cb2acf1f966da1fe5c19c77a70282b25f4c5f
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
-  skip: true  # [py2k]
+  number: 1000
+  script: "{{ PYTHON }} -m pip install . -vvv"
+  skip: True  # [py<35]
 
 requirements:
   build:
-    - python >=3.5.3
```

Comment:
This recipe needs "update-cb3" too. Also, can it be `noarch`?

Reply:
yeah, it appears it can! Let me try, sorry for not noticing that...

Reply:
thanks a lot @ocefpaf, it works, sorry for bothering you (and core) for such a silly mistake...

Reply:
No problem.